### PR TITLE
Parent li should not contain active class. Fixes #135

### LINKED ISSUE #135: Parent li should not contain active class
From the Tabs widget, `<li class="nav-item active">` should not contain the `active` class. See [Tabs](https://getbootstrap.com/docs/4.3/components/navs/#tabs).

### PR DESCRIPTION
**Pull request by Thoulah**
| Q             | A
| ------------- | ---
| Is bugfix?    | yes
| New feature?  | no
| Breaks BC?    | no
| Tests pass?   | yes
| Fixed issues  | #135

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,7 @@ Yii Framework 2 bootstrap4 extension Change Log
 
 - Bug #131: Fixed element with role tab must be inside role tablist (simialbi)
 - Enh #129: Added `Modal::SIZE_EXTRA_LARGE` constant (shoomlix)
+- Bug #135: Parent li should not contain active class (Thoulah)
```
Comment (samdark) [inline CHANGELOG.md]:
```suggestion
- Bug #135: Parent `li` should not contain active class (Thoulah)
```

File: src/Nav.php [hunk 2]
```diff
@@ -84,6 +84,10 @@ class Nav extends Widget
     /**
      * @var bool whether to activate parent menu items when one of the corresponding child menu items is active.
      */
+    public $activateContainer = true;
+    /**
+     * @var bool whether to activate the container when the menu item is active.
```
Comment (samdark) [inline src/Nav.php]: Seems descriptions aren't for their corresponding properties.

Comment (Thoulah) [reply src/Nav.php]: My bad, fixed now

File: src/Nav.php [hunk 3]
```diff
@@ -81,6 +81,10 @@ class Nav extends Widget
      * @see isItemActive
      */
     public $activateItems = true;
+    /**
+     * @var bool whether to activate the container when the menu item is active.
```
Comment (samdark) [inline src/Nav.php]: Need `@since 2.0.3`.

Comment (Thoulah) [reply src/Nav.php]: Done

**Comment by samdark** [comment]
Merged. Thank you!

**Pull request merged**